src/NextGen/DonationForm/Controllers/DonationFormViewController.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Give\NextGen\DonationForm\Controllers;
+
+use Give\Framework\EnqueueScript;
+use Give\NextGen\DonationForm\DataTransferObjects\DonationFormViewRouteData;
+use Give\NextGen\DonationForm\Repositories\DonationFormRepository;
+use Give\NextGen\DonationForm\ViewModels\DonationFormViewModel;
+use Give\NextGen\Framework\FormTemplates\Registrars\FormTemplateRegistrar;
+
+class DonationFormViewController {
+    /**
+     * @unreleased
+     */
+    public function show(DonationFormViewRouteData $data): string
+    {
+        $viewModel = new DonationFormViewModel($data->formId);
+
+        ob_start();
+        $this->head();
+        $this->enqueueScripts($data->formId, $data->formTemplateId);
+        ?>
+
+        <script>window.giveNextGenExports = <?= wp_json_encode($viewModel->exports()) ?>;</script>
+
+        <div id="root-give-next-gen-donation-form-block"></div>
+
+        <?php
+        wp_print_footer_scripts();
+        echo ob_get_clean();
+
+        exit();
+    }
+
+    /**
+     * @unreleased
+     *
+     * @return void
+     */
+    private function head()
+    {
+        global $wp_scripts, $wp_styles;
+        wp_head();
```

Comment:
What is it we're trying to preserve in `wp_head()` that we want in the iframe? We're clearly removing every script and style. Same thing with the footer. There would be meta and such in here I'm not sure we want to keep. But I'm curious.

Reply:
For now, just removing all the baggage from WP.  Then whatever we enqueue (using WP enqueue) on our own will end up in the head & footer.

Reply:
From our call: this is going to be used to preserve the WP enqueuing system.